### PR TITLE
Oracle UI contract integration

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
# Contract Interaction

fetching the contract details and displaying them on the frontend

Added Subpages.

fixes https://github.com/EveripediaNetwork/issues/issues/917
fixes https://github.com/EveripediaNetwork/issues/issues/873

### CODE REVIEW

**Comment by Adebesin-Cell** [comment]
Awesome 👏 .

**Comment by Emmanueldmlr** [comment]
@Adebesin-Cell @OleanjiKingCode  for march madness contract, it returns an IPFS hash (e.g, the contract returns QmcJ9AQPjqDbJJ8KDMiaAF8KXRGwAJx7eWf3WNGsEijKBQ as the hash) to get the data, you have to fetch it directly from ipfs using the format https://gateway.pinata.cloud/ipfs/QmcJ9AQPjqDbJJ8KDMiaAF8KXRGwAJx7eWf3WNGsEijKBQ

**Comment by Emmanueldmlr** [comment]
for now, Ap election integration is pending because I don't have access to the existing codebase...Aside from that, you both have everything to finish the task. T for Thanks ✌🏿

**Comment by Adebesin-Cell** [comment]
@Emmanueldmlr okay, thanks.

**Comment by Softdev1** [comment]
what access do you need so that i can help you get it @Emmanueldmlr

**Comment by Emmanueldmlr** [comment]
@Simplypheyie96, can you review the UI implementation also?

**Review by Simplypheyie96 — CHANGES_REQUESTED** [review]
The press page needs to be redone.
the height of the cards are not meant to be equal, so it implemented like that is creating a disunity in the UI. the card contents are not in sync
image

also make the links on the press page clickable, so it will take the users directly to the site. It should be underlined on hover.
image

is there a way for this to be like the design and not this blue?
image

**Comment by Simplypheyie96** [comment]
@Adebesin-Cell I forgot to mention this. some buttons are missing on the header in the sm breakpoint
image

**Comment by Simplypheyie96** [comment]
> @Adebesin-Cell I forgot to mention this. some buttons are missing on the header in the sm breakpoint image But it seems like the design won't translate on implementation as the space is small compared to the one on figma, so I will work out something else and let you know for sm bp

let's use this for changes on the light and dark mode. but still fix for the network change button @Adebesin-Cell

**Comment by Adebesin-Cell** [comment]
I’ll push the fixes soon.

**Review by Simplypheyie96 — APPROVED** [review]
Lots of fixes needed. But i will approve for now for today

File: src/components/Oracles/MarchMadness/InfoView.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&quot;
+          <Box as="span" color="green.500">
+            outputs
+          </Box>
+          &quot;: [
+          <br />
+          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;]
+          <br /> &#125;]
+        </Box>
+      </Box> */}
```
Comment (Royal-lobster) [inline src/components/Oracles/MarchMadness/InfoView.tsx:129]:
```suggestion
```
we can remove this, if needed we can go back to previous commits to get back this information.

File: src/components/Oracles/MarchMadness/PressViewCard.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+const BROKEN_IMAGE = '/images/broken.png'
+
+type PressViewCardProps = { id: number } & PressJsonprops
+
+const MarchMadnessPressViewCard = (props: PressViewCardProps) => {
+  const { image, link, text, id, title } = props
+
+  const index = id + 1
+
+  console.log(index)
```
Comment (Royal-lobster) [inline src/components/Oracles/MarchMadness/PressViewCard.tsx:14]:
```suggestion
```

File: src/components/Layout/Footer/Footer.tsx [hunk 3]
```diff
@@ -1,109 +1,8 @@
...
-                  </MenuList>
-                </Menu>
-              </Box>
-            </HStack>
-          </Stack>
-        </SimpleGrid>
-      </Container>
-    </Box>
-  )
+  return <Box>&nbsp;</Box>
```
Comment (Royal-lobster) [inline src/components/Layout/Footer/Footer.tsx:5]: Are we going to have footer for the page ? if not we can remove this file.

Comment (Adebesin-Cell) [reply src/components/Layout/Footer/Footer.tsx:5]: Yeah, we’ll have a footer later.

Comment (Royal-lobster) [reply src/components/Layout/Footer/Footer.tsx:5]: Cool. then its fine.

File: src/components/Oracles/MarchMadness/InfoView.tsx (hunk 1 above)
Comment (Adebesin-Cell) [reply src/components/Oracles/MarchMadness/InfoView.tsx:129]: Alright, I’ll fix that now.

Comment (Royal-lobster) [reply src/components/Oracles/MarchMadness/InfoView.tsx:129]: You can click on commit suggestion, it should add the fix

**Review by Royal-lobster — APPROVED** [review]

File: src/components/Oracles/MarchMadness/PressViewCard.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+import { PressJsonprops } from '@/data/MarchMadnessPressData'
+
+const BROKEN_IMAGE = '/images/broken.png'
+
+type PressViewCardProps = { id: number } & PressJsonprops
+
+const MarchMadnessPressViewCard = (props: PressViewCardProps) => {
+  const { image, link, text, id, title } = props
+
+  const index = id + 1
```
Comment (Royal-lobster) [inline src/components/Oracles/MarchMadness/PressViewCard.tsx:12]:
```suggestion
```

**Comment by Softdev1** [comment]
this page looks cool, we would still some iterations, on the page if there is possible .. well done guys @OleanjiKingCode  @Adebesin-Cell

**Pull request merged**